internal/querynodev2/segments/segment_loader.go

```diff
@@ -818,7 +818,7 @@ func JoinIDPath(ids ...UniqueID) string {
 	return path.Join(idStr...)
 }
 
-func GetStorageSizeByIndexInfo(indexInfo *querypb.FieldIndexInfo) (uint64, uint64, error) {
+func GetIndexStorageUsage(indexInfo *querypb.FieldIndexInfo) (uint64, uint64, error) {
```

Comment:
Not accurate as well, it returns disk & memory usage, maybe `ResourceUsage` is better?